src/core/dxf/qgsdxfexport.cpp

```diff
@@ -150,8 +150,9 @@ void QgsDxfExport::writeGroup( const QColor &color, int exactMatchCode, int rgbC
 
   int c = ( color.red() & 0xff ) * 0x10000 + ( color.green() & 0xff ) * 0x100 + ( color.blue() & 0xff );
   writeGroup( rgbCode, c );
-  if ( transparencyCode != -1 && color.alpha() < 255 )
-    writeGroup( transparencyCode, 0x2000000 | color.alpha() );
+  // transparency group code 440 leads to ACAD error (#34841)
```

Comment:
Is this a QGIS issue or an ACAD issue?

Reply:
I was wondering the same. I think it's disappointing to limit this functionality because of a 3rd party bug. What about adding an option in the UI for "Safe export for maximum compatibility"?

Reply:
Last version of AutoDesc's software "TrueViewer..." crashes on this code. Some old versions of AutoCAD can't process it too. I think to add option to UI is best way.

Reply:
It's possible that this code is indeed broken (== it actually is a QGIS issue and we should solve it).
Are you able to check what autodesk itself produces for transparency?